services/discordService.js

```diff
@@ -85,10 +85,27 @@ const setUserDiscordNickname = async (userName, discordId) => {
   }
 };
 
+const markUserVerified = async (discordId) => {
+  try {
+    const unverifiedRoleId = config.get("discordUnverifiedRoleId");
+
```

Comment:
Suggestion: Instead of getting unverifiedRoleId at each function call, can we not just get the value of this once at the top of our file?